src/server/jobs/update-musicres.js

```diff
@@ -23,14 +23,20 @@ async function main() {
     let totalNumEntries = 0;
     for (const course of res.data.courses) {
       const { term, srs, works } = course;
-      const shortName = await RegistrarService.getShortname(term, srs);
-      if (!shortName) {
+      const shortnameResponse = await RegistrarService.getShortname(term, srs);
+      if (!shortnameResponse) {
         logger.warn(`${course.term}-${course.srs} does not have shortname`);
       }
+      const shortname = shortnameResponse ? shortnameResponse.shortname : null;
```

Comment:
@chern Do you need all this logic? Can't you make RegistrarService.getShortname set these fields to null if they don't exist?